aloe/registry.py

```diff
@@ -449,3 +454,111 @@ def decorator(self, function, **kwargs):
 around = CallbackDecorator(CALLBACK_REGISTRY, 'around')
 before = CallbackDecorator(CALLBACK_REGISTRY, 'before')
 # pylint:enable=invalid-name
+
+
+class ExtendedStep(object):
```

Comment:
I'm still looking through the implementation, but I'd really prefer to do what Behave does and register a new sentence parser instead of throwing everything into regular expressions.